### ISSUE TITLE
unclear parts of 6.2

### ISSUE THREAD

**Issue by RufaelDev** [Dash-Industry-Forum/Ingest#160]
6.2.2 media ingest connection is not defined 
6.2.5 preceding requirements, if a track failed only resending the header should be sufficient

**Comment by almson** [comment]
This.

The entire concept of CMAF-Ingest is not explained and doesn't make sense. What is an ingest connection? When is it initiated and when does it end? Is it a TCP connection? Is it a higher-level concept above the HTTP layer? How are we supposed to implement a robust receiver if the *connection* is not defined? The spec says in the damn introduction, "First of all, the choice for the data transmission protocol, and connection establishing and tearing down **are important**."

**Comment by RufaelDev** [comment]
yes, this point was raised, in fact the ingest should be stateless, before it was based on a long running post based on smooth streaming ingest from microsoft and teardown was important in that case, currently it is short running posts of segments,  some tools and sample receiver are here: https://github.com/unifiedstreaming/fmp4-ingest overall this spec targets encoder/origin interop and in practice each encoder does implement cmaf ingest slightly different using mpd/m3u8 and posting of segments (sometimes the segments are produced in slightly different way). The most important is that you can receive the segments and append them back into cmaf tracks.  This issue should still be addressed in the document.
